capstone-rs/src/capstone.rs

```diff
@@ -365,6 +365,44 @@ impl Capstone {
         }
     }
 
+    /// Get the registers are which are read to and written to, in that order.
+    pub fn regs_access(&self, insn: &Insn) -> Option<CsResult<(Vec<RegId>, Vec<RegId>)>> {
```

Comment:
Return a `CsResult<...>` instead of an `Option<CsResult<...>>`.

You can return `Err(Error::DetailOff)` if the "full" feature is not enabled.